domains/auth/Auth.API/API/Repository/UserStorage.cs

```diff
@@ -0,0 +1,9 @@
+using API.Models;
+
+namespace API.Repository;
+
+public class UserStorage : IUserStorage
+{
+
+    public Task<User?> UserByOidcReferences(string subject, string provider) => Task.FromResult<User?>(null);  //FIXME implement this
```

Comment:
Is the comment needed?
I think if we have //TODO/FIXME there should be a link to a story fixing it